src/content/basics/nodepools/index.md

```diff
@@ -113,7 +113,40 @@ Pay close attention to the workloads being rescheduled on other nodes once nodes
 
 See the [`gsctl delete nodepool`](/reference/gsctl/delete-nodepool/) reference for how to delete a node pool using the CLI.
 
-## Node pools and the Giant Swarm API
+## Instance distribution
+
+Node pools can contain a mix of [on-demand](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-on-demand-instances.html) and [spot instances](https://aws.amazon.com/ec2/spot/) that will allow you to optimize your cost.
+
+There are two parameters that will allow you to configure which instances are going to be used:
+
+- *On-demand base capacity*: controls how much of the initial capacity is made up of on-demand instances.
+
+- *Spot instance percentage above base capacity*: controls the percentage of spot instances to be used for worker nodes beyond the number of *on-demand base capacity*.
+
+### Examples
+
+The following table shows four examples to illustrate how different settings of spot instance percentage and on-demand base capacity influence the outcome.
+
+| On-demand base capacity | Spot instance percentage | Total Instances  | On-Demand Instances| Spot Instances
+|:-:|:-:|:-:|:-:|:-:|
+| 0 | 0 % | 50 | 50 | 0
+| 0 | 100 % | 50 | 0 | 50
+| 10 | 50 % | 50 | 30 | 20
+| 10 | 100 % | 50 | 10 | 40
+
+## Using similar instance types {#similar-instance-types}
+
+Starting with release v{{% first_aws_spotinstances_version %}} on AWS you can activate the use of similar instance types per node pool. With this setting active, your node pool can use instance types that are nearly identical to the one you selected. For example, if you select `m5.xlarge`, the node pool can also use `m4.xlarge`.
+
+Using multiple instance types in a node pool has some benefits:
+
+- Together with spot instances, using multiple instance type allows better price optimization. Popular instance types tend to have more price adjustments. Allowing older-generation instance types that are less popular tends to result in lower costs and fewer interruptions.
+
+- Even without spot instances, AWS has a limited number of instances per type in each Availability Zone. I can happen that your selected instance type is temporarily out of stock just in the moment you are in demand of more worker nodes. Allowing the node pool to use multiple instance types reduces this risk and increases the likelihood that your node pool can grow when in need.
```

Comment:
```suggestion
- Even without spot instances, AWS has a limited number of instances per type in each Availability Zone. It can happen that your selected instance type is temporarily out of stock just in the moment you are in demand of more worker nodes. Allowing the node pool to use multiple instance types reduces this risk and increases the likelihood that your node pool can grow when in need.
```